witchcraft-server/src/logging/mod.rs

```diff
@@ -59,10 +65,44 @@ pub(crate) async fn init(
     let request_logger = logger::appender(install, metrics, hooks).await?;
     let audit_logger = logger::appender(install, metrics, hooks).await?;
 
+    AUDIT_LOGGER.fill(audit_logger.clone()).map_err(|_| {
+        Error::internal_safe("Audit logger already initialized")
+    })?;
     cleanup::cleanup_logs().await;
 
     Ok(Loggers {
         request_logger,
         audit_logger,
     })
 }
+
+/// Write the provided v3 audit log entry to the audit log using the global audit logger.
+/// Returns an error if the global audit logger is not initialized.
+///
+/// The returned future completes once the audit log has been successfully written.
+pub async fn audit_v3_log(AuditLogEntry(v3_log): AuditLogEntry) -> Result<(), Error> {
```

Comment:
The name/docs shouldn't refer to v3 audit logs now that we're taking an audit log entry.

It's more common to have the argument be just `entry: AuditLogEntry` and then use `entry.0` below to get at the inner value.